internal/cmd/auth_createapitokens.go

```diff
@@ -27,7 +29,14 @@ var createApiTokensCmd = &cobra.Command{
 			return err
 		}
 
-		tokenName := args[0]
+		tokenName := strings.TrimSpace(args[0])
+
+		validNameRegex := regexp.MustCompile("^[a-zA-Z0-9_-]+$")
```

Comment:
It doesn't quite apply here, but in future uses of the `regexp` package please:
- Use `regexp.MustCompile` in a place that is clearly called once at the beginning of the program life (e.g. at the package level) and reused.
- Otherwise use `regexp.Compile`.